### PR TITLE
Installation readme

### PR DESCRIPTION
**Pull request by datailluminations**
Added a draft of Installation.md per Item #14 in Homestretch Google doc.

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
```
Comment (iEatYanYans) [inline .idea/modules.xml:3]: i dont think we should commit this file

File: .idea/gradvek.iml [hunk 2]
```diff
@@ -2,10 +2,7 @@
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/springdb/src/main/java" isTestSource="false" />
-      <sourceFolder url="file://$MODULE_DIR$/springdb/src/test/java" isTestSource="true" />
-    </content>
+    <content url="file://$MODULE_DIR$" />
     <orderEntry type="inheritedJdk" />
```
Comment (iEatYanYans) [inline .idea/gradvek.iml:6]: do we need this + the other file?

Comment (datailluminations) [reply .idea/gradvek.iml:6]: Hmm.. I'm not sure how this file got there? LOL. I can delete if we are not using it. Perhaps, VSCode generated these extraneous file.

Comment (iEatYanYans) [reply .idea/gradvek.iml:6]: yeah it looks like some kind of metadata file lol

**Review by iEatYanYans — APPROVED** [review]
lgtm

**Pull request merged**